client/src/app/services/posts.service.ts

```diff
@@ -1,5 +1,4 @@
 import { Injectable } from '@angular/core';
-
```

Comment:
Revert this change. There should be a space between "their" (the angular team) and "our" imports ([link to the style guide](https://angular.io/styleguide#!#03-06)).